src/core/tsi/alts/handshaker/alts_tsi_handshaker.h

```diff
@@ -60,6 +62,9 @@ typedef struct alts_tsi_handshaker alts_tsi_handshaker;
 ///  method.
 ///- user_specified_max_frame_size: Determines the maximum frame size used by
 ///  frame protector that is specified via user. If unspecified, the value is 0.
+///- preferred_transport_protocols: a comma separated list of preferred
+///  transport protocols to be negotiated for this connection. If empty, no
+///  negotiation will occur and the transport protocol will default to http/2.
```

Comment:
IIRC, we decided that we will not set a default. If that is indeed the case, please remove the "and the transport protocol will default to http/2" bit of the sentence.